src/gcp_scanner/scanner.py

```diff
@@ -475,6 +477,31 @@ def gke_client_for_credentials(
     credentials=credentials)
 
 
+def get_sa_details_from_key_files(key_path):
```

Comment:
Looks good. Could you please add a test for this function? One that fails and one that succeed.

Reply:
done